src/rosdiscover/models/topic_tools_mux.py

```diff
@@ -21,3 +21,22 @@ def mux(c):
     c.pub(topics.outopic, "any")
     for in_topic in topics.intopic:
         c.sub(in_topic, "any")
+
+
+@model("topic_tools", "relay")
+def relay(c):
+    c.read("~unreliable", False)
+    c.read('~lazy', False)
+    c.read('~stealth', False)
+    # TODO: Chris: how should we treat this? I think this means a topic is created
+    # that by default is the same is intopic, but could be set by a parameter
+    # http://wiki.ros.org/topic_tools/relay
+    c.read('~monitor_topic', 'intopic')
```

Comment:
`topic_name = c.read(....)`

Reply:
Done. I am not sure of the weird semantics for this: http://wiki.ros.org/topic_tools/relay. But it does something now.